addons/mail/models/mail_render_mixin.py

```diff
@@ -422,6 +423,7 @@ def _render_template_jinja(self, template_txt, model, res_ids,
 
         # prepare template variables
         variables = self._render_jinja_eval_context()
+        variables['is_html_empty'] = is_html_empty
```

Comment:
Should not be necessary as it calls common eval context since a few days, could you check ?

Reply:
Yes, you are right. I have adapted the changes.